index.html

```diff
@@ -9,11 +9,12 @@
 <body>
   <h1>Awesome Books</h1>
   <div id="book-list"></div>
-  <form id="add-book-form">
-    <input type="text" id="title" name="title" placeholder="Title" required><br><br>
-    <input type="text" id="author" name="author" placeholder="Author" required><br><br>
-    <button type="submit">Add</button>
-  </form>
+
+    <form id="add-book-form">
+      <input type="text" id="title-input" placeholder="Title" required>
+      <input type="text" id="author-input" placeholder="Author" required>
+      <button type="submit">Add Book</button>
+    </form>
```

Comment:
- [ ] The required change has not been implemented. Kindly attend to it and make your design looks like the wireframe as required from the  [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes_v1_1.md).

### Your Design
![your-design](https://github.com/Jessekay/AwesomeBooks/assets/26597432/7186b59c-50db-4608-bd11-b8dc97117166)

### Wireframe
![wireframe](https://github.com/Jessekay/AwesomeBooks/assets/26597432/7b0796df-f112-4d2a-a295-824e59272f92)

Reply:
Hi sir thanks for your feedback
DO  mean we should produce the same design or we can use our design and make it similar to this?